packages/web-components/fast-foundation/src/design-system-provider/design-system-provider.ts

```diff
@@ -477,7 +477,12 @@ export class DesignSystemProvider extends FASTElement
     private syncDesignSystemWithProvider(): void {
         if (this.provider) {
             const designProperties = this.designSystemProperties;
```

Comment:
Looks like this isn't used on line 482, you could just remove the variable and refer to `this.designSystemProperties`.